### DISCUSSION TITLE
Is it possible to run SDRTrunk with params from Windows CLI?

### DISCUSSION THREAD

**Discussion by erezcarmel** [DSheirer/sdrtrunk#1788] [Q&A] [answered]
I'm willing to run a specific playlist/channel from CLI on Windows. Is it possible?
or the headless option runs only on Linux?

**Comment by DSheirer** [comment, marked as answer]
Currently, sdrtrunk will default to the last-used playlist.  At some future point, I"ll probably add support for this, but there's a lot of other things on my plate between now and then.

**Comment by erezcarmel** [comment]
@DSheirer Thank you for the quick answer. Keep up the great work on the project.
